flink-python/pyflink/datastream/stream_execution_environment.py

```diff
@@ -553,6 +553,34 @@ def set_python_executable(self, python_exec: str):
             .getEnvironmentConfig(self._j_stream_execution_environment)
         env_config.setString(jvm.PythonOptions.PYTHON_EXECUTABLE.key(), python_exec)
 
+    def add_jars(self, jars_path: str):
+        """
+        Adds a list of jar files that contain the user-defined function (UDF) classes and all
```

Comment:
I think `add_jars` is not only used in udf situations, maybe you need to change the annotation